src/libaktualizr/package_manager/ostreemanager.cc

```diff
@@ -272,8 +272,8 @@ Uptane::Target OstreeManager::getCurrent() const {
   storage_->loadPrimaryInstalledVersions(&installed_versions, nullptr, nullptr);
 
   // Version should be in installed versions
-  std::vector<Uptane::Target>::iterator it;
-  for (it = installed_versions.begin(); it != installed_versions.end(); it++) {
+  std::vector<Uptane::Target>::reverse_iterator it;
+  for (it = installed_versions.rbegin(); it != installed_versions.rend(); it++) {
```

Comment:
Probably worth a comment to explain the reverse_iterator preference.

Reply:
good point.